trainee_template/react_series.amlg

```diff
@@ -64,8 +64,8 @@
 
 		(declare (assoc
 			warnings (assoc)
-			output_influential_cases (get details "influential_cases")
-			output_cap (get details "categorical_action_probabilities")
+			requested_details (filter (lambda (get details (current_value))) (indices details))
```

Comment:
i'm confused as to what this filter is doing exactly? seems like since 'details' is an assoc, you should simplify this to this instead:
`requested_details (indices (filter (lambda (current_value)) details))`

Reply:
you're completely right. Thanks